starknet-providers/src/any.rs

```diff
@@ -667,4 +668,45 @@ impl Provider for AnyProvider {
             }
         }
     }
+
+    async fn batch_requests<I, P>(
+        &self,
+        requests: I,
+    ) -> Result<Vec<serde_json::Value>, ProviderError>
+    where
+        I: IntoIterator<Item = (JsonRpcMethod, P)> + Send + Sync,
+        P: Serialize + Send + Sync,
+    {
+        match self {
+            Self::JsonRpcHttp(inner) => {
+                <JsonRpcClient<HttpTransport> as Provider>::batch_requests(inner, requests).await
+            }
+            Self::SequencerGateway(inner) => {
+                <SequencerGatewayProvider as Provider>::batch_requests(inner, requests).await
+            }
+        }
+    }
+    async fn get_block_with_tx_hashes_batch<B>(
```

Comment:
fmt nit: can we have a space between the two functions?

Reply:
Fixed.